wordpress-mobile/WordPress-iOS
# Comment edit: use updated values for display

Ref: #17000

This fixes an issue where updated strings were overwritten with the original strings when the cells were scrolled out of view.

To test:
- Go to My Site > Comments > Comment details > Edit.
- Change any of the top three fields (Name, Email Address, Web Address).
- Scroll so the updated field is out of view, and scroll back to redisplay.
  - You may need to enter a lot of text in the `Comment` field, and/or increase text size.
- Verify the updated text persists, and is not reverted back to the original.

---
**Before:**

https://user-images.githubusercontent.com/1816888/131918761-2465ed2b-bfc0-4cca-8e4f-662d2bdf1e79.mp4

**After:**

https://user-images.githubusercontent.com/1816888/131918538-ef939482-fb2d-4b03-beeb-a6f050c46c60.mp4

---
## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
